cli/tests/environment-managed.bats

```diff
@@ -482,12 +482,9 @@ EOF
   make_empty_remote_env
   "$FLOX_BIN" install hello
 
-  # TODO: flox will set HOME if it doesn't match the home of the user with
-  # current euid. I'm not sure if we should change that, but for now just set
-  # USER to REAL_USER.
-  FLOX_SHELL=bash USER="$REAL_USER" run -0 expect "$TESTS_DIR/activate/interactive-hello.exp" "$PROJECT_DIR"
-  assert_output --regexp "$FLOX_CACHE_DIR/run/owner/.+\..+\..+/bin/hello"
-  refute_output "not found"
+  run "$FLOX_BIN" activate --dir "$PROJECT_DIR" -- bash -c "command -v hello || which hello || type -P hello"
```

Comment:
question: I get why we are not (only) using `which`, but why use three approaches when `command -v` should work as a builtin of bash?

Reply:
Good point. Updated with:

> These were copied from the previous `expect` script which is executed
> with several different shells that may not include the `command`
>builtin. We don't need to do that now that we're only running these with
> `bash`.